check-factors/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+
+  <script>
+    function checkFactors(array, number){
+      for (var i=0; i<array.length; i++){
+        var num = number/array[i];
+        if(Number.isInteger(num)){
```

Comment:
Hey Jay, this will work for some submissions, but you have to make sure to check all of the values before you return! This will hit a return statement after checking only one value!

Reply:
I will check again. Thanks Bret!